cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs

```diff
@@ -392,6 +428,31 @@ runGenesisCreateTestNetDataCmd
     utxoKeysDir = outputDir </> "utxo-keys"
     poolsDir = outputDir </> "pools-keys"
     stakeDelegatorsDir = outputDir </> "stake-delegators"
+    mkPoolDir idx = poolsDir </> ("pool" <> show idx)
+    byronPoolNumber = max 1 numPools -- byron genesis creation needs a >= 1 number of pools
+
+    -- All arbitrary values come from cardano-testnet
+    mkByronGenesisParameters actualNetworkWord32 byronGenesisFp shelleyGenesis =
```

Comment:
All of the Byron functionality should be factored out on its own as much as possible.